nilearn/glm/contrasts.py

```diff
@@ -235,9 +236,11 @@ def stat(self, baseline=0.0):
         self.stat_ = stat.ravel()
         return self.stat_
 
+
     def p_value(self, baseline=0.0):
-        """Return a parametric estimate of the p-value associated
-        with the null hypothesis: (H0) 'contrast equals baseline'
+        """Return a parametric estimate using the survival function of the
```

Comment:
maybe "return a parametric estimate of the p-value associated [...], using the survival function" ? otherwise it reads like "... the survival function of the p-value"

Reply:
OK, fixed!